doc/nrf/app_dev/ap_protect/index.rst

```diff
@@ -0,0 +1,156 @@
+   * - nRF5340
+     - Hardware and software; also supports Secure AP-Protect, given support for different :ref:`app_boards_spe_nspe`
+     - `AP-Protect for nRF5340`_
+   * - nRF52840
+     - Hardware; Hardware and software
+     - `AP-Protect for nRF52840`_
+   * - nRF52833
+     - Hardware; Hardware and software
+     - `AP-Protect for nRF52833`_
+   * - nRF52832
+     - Hardware; Hardware and software
+     - `AP-Protect for nRF52832`_
+   * - nRF52820
+     - Hardware; Hardware and software
+     - `AP-Protect for nRF52820`_
+   * - nRF52811
+     - Hardware; Hardware and software
+     - `AP-Protect for nRF52811`_
+   * - nRF52810
+     - Hardware; Hardware and software
+     - `AP-Protect for nRF52810`_
+   * - nRF52805
+     - Hardware; Hardware and software
+     - `AP-Protect for nRF52805`_
+
+.. note::
+    The SoCs that support `ARM TrustZone`_ (nRF5340 and nRF9160) implement two AP-Protect systems: AP-Protect and Secure AP-Protect.
+    While AP-Protect blocks access to all CPU registers and memories, Secure AP-Protect limits access to the CPU to only non-secure accesses.
+    This means that the CPU is entirely unavailable while it is running the code in the Secure Processing Environment, and only non-secure registers and address-mapped resources can be accessed.
+
+.. _app_approtect_ncs:
+
+Configuration overview in the |NCS|
+***********************************
+
+Based on the available implementation types, you can configure the access port protection mechanism in the |NCS| to one of the following states:
+
+.. list-table:: AP-Protect states
+   :header-rows: 1
+
+   * - AP-Protect state
+     - Related Kconfig option in the |NCS|
+     - Description
+     - AP-Protect implementation type
+   * - Locked
+     - :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK` (:kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_LOCK` for Secure AP-Protect)
+     - In this state, CPU writes to enable and lock AP-Protect. UICR is not modified.
+     - Hardware and software
+   * - Authenticated
+     - :kconfig:option:`CONFIG_NRF_APPROTECT_USER_HANDLING` (:kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_USER_HANDLING` for Secure AP-Protect)
+     - In this state, AP-Protect is left enabled and it is up to the user handler to unlock the device if needed.
+     - Hardware and software
+   * - Open
+     - :kconfig:option:`CONFIG_NRF_APPROTECT_USE_UICR` (:kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_USE_UICR` for Secure AP-Protect)
+     - In this state, AP-Protect follows the UICR register. If the UICR is open, the AP-Protect will be disabled.
+     - Hardware; hardware and software
+
+.. _app_approtect_ncs_lock:
+
+Enabling AP-Protect with :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK`
+====================================================================
+
+Setting the :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK` Kconfig option to ``y`` and compiling the firmware is enough to enable the access port protection mechanism for SoCs of the nRF52 and nRF53 Series.
```

Comment:
Perhaps it would be good to specify here that for the nRF52 Series this only applies to SoCs featuring the hardware and software type of AP-Protect, to avoid confusion. This conclusion can be derived from the table above, but I'm afraid it might not be obvious.